blakserv/main.c

```diff
@@ -99,6 +99,11 @@ void MainServer()
 	InitTable();
 	AddBuiltInDLlist();
 	
+	/* Initialize webhook system - controlled by [Webhook] Enabled config setting
+	   Multiple servers are automatically supported - each server claims an available pipe
+	   from the pool of 10 pipes created by external webhook listeners */
+	InitWebhooks(NULL);

```

Comment:
Since NULL is always passed for the prefix, what is the purpose of this argument?